### PR TITLE
Image permissions

### PR DESCRIPTION
**Pull request by djredhand**

### CODE REVIEW

File: uelc/main/views.py [hunk 1]
```diff
@@ -488,14 +488,26 @@ def post(self, request):
         user.profile.profile_type = profile
         if not profile == "group_user":
             user.is_staff = True
+            self.set_image_upload_permissions(user)
         else:
             user.is_staff = False
+            self.set_image_upload_permissions(user)
```
Comment (sdreher) [inline uelc/main/views.py]: self.set_image_upload_permissions(user) could be moved out of the if/else statement and called just once.

File: uelc/main/views.py [hunk 2]
```diff
@@ -488,14 +488,26 @@ def post(self, request):
...
 
+    def set_image_upload_permissions(self, user):
+        permission_set = Permission.objects.filter(
+            content_type__name="image upload item")
+        for perm in permission_set:
+            if user.is_staff:
+                user.user_permissions.add(perm.pk)
+            else:
+                user.user_permissions.remove(perm.pk)
+        user.save()
```
Comment (sdreher) [inline uelc/main/views.py]: user.save() is not needed here.

**Comment by sdreher** [comment]
Looks like you merged this branch into https://github.com/ccnmtl/uelc/pull/131. Do you want to close this one?

**Comment by djredhand** [comment]
Yep, I did. That's why I was wanting to merge myself. Thanks for the comments!

**Pull request closed without merging**